node-notes-cli/app.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const json = require('./data.json');
+
+const keys = Object.keys(json.notes);
+const lastKey = keys[keys.length - 1];
+
+if (process.argv[2] === 'read') {
+  console.log(JSON.stringify(json.notes, null, 2));
```

Comment:
Double-check the example output of the GIF in the instructions, it is not a JSON string.